cmd/main.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
We plan try to merge this webhook to upstream in the nearest future. Until it is done I'll replace the copyrights in a separate commit which could be easily reverted in future.